R/import.R

```diff
@@ -1,6 +1,6 @@
 mintr_db_download <- function(path, quiet = FALSE) {
   version <- mintr_data_version()
-  root <- "https://mrcdata.dide.ic.ac.uk/mint"
+  root <- "https://github.com/mrc-ide/mintr/releases/download/data-20230208"
   url <- sprintf("%s/%s.tar", root, version)
```

Comment:
```suggestion
  root <- "https://github.com/mrc-ide/mintr/releases/download/"
  url <- sprintf("%s/data-%s/%s.tar", root, version, version)
```

though I imagine we'll update the pattern at some point, this is closer to the intent I think